### DISCUSSION TITLE
How do you use a DLL or Assembly in the AfterInstall of Wixsharp?

### DISCUSSION THREAD

**Discussion by kplatter** [oleg-shilo/wixsharp#1470] [General]
I need to be able to query a database and would like to use Dapper (an assembly I installed thru nuget)...  When I run the install I get

Could not load file or assembly 'Dapper, Version=2.0.0.0...

How do I make assemblies available to use in the setup?

**Comment by oleg-shilo** [comment]
```C#
project.DefaultRefAssemblies.Add(typeof(SomeDapperType).Assembly.Location);
```

**Comment by kplatter** [comment]
Ty !!!!  Is there a sample/documentation of this that I missed?

**Comment by oleg-shilo** [comment]
https://github.com/oleg-shilo/wixsharp/wiki/Deployment-scenarios#managed-custom-actions
![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/6fd596da-a64a-473a-9826-ad9af2a3bb10)

For WixSharp the documentation comes in a few forms:
- Wiki. Obvious. It's here, in this repo.
- Code samples. The most valuable knowledge asset. With WixSharp I made a deliberate choice not to provide samples as a collection of code snippets. For complex topics like WiX this approach is just not good enough. Instead. WixSharp provides a set of **complete,** fully functional samples as a (script library)[https://github.com/oleg-shilo/wixsharp/tree/master/Source/src/WixSharp.Samples/Wix%23%20Samples]. Samples are making close to 90% of the whole codebase.
- And of course intellisense.
